### PR TITLE
Create Outlook.Folder.GetOwner

### PR DESCRIPTION
**Pull request by emyoshi**
Creating a page for the new Folder.GetOwner method available in the April 2024 fork of Office for Outlook Desktop. I think this is complete except perhaps the keywords, f1_keywords, and ms_assetid fields. I'm not sure how those work.

### CODE REVIEW

File: Outlook.Folder.GetOwner [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Folder.GetOwner method (Outlook)
+
+// how to do comments?? I'm not sure about this block
```
Comment (lindalu-MSFT) [inline Outlook.Folder.GetOwner:4]:
```suggestion

```

File: Outlook.Folder.GetOwner [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+keywords: vbaol11.chm3556
+f1_keywords:
+- vbaol11.chm3556
+api_name:
+- Outlook.Folder.GetOrganizer
+ms.assetid: ??
```
Comment (lindalu-MSFT) [inline Outlook.Folder.GetOwner:10]:
```suggestion
ms.assetid: e728cef8-170f-4286-bf3b-6a1acad6cc22
```
Just generate a GUID from any online GUID generator.

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]

**Review by lindalu-MSFT — APPROVED** [review]

**Comment by lindalu-MSFT** [comment]
@emyoshi Hi Erika, since VBA is in maintenance, we won't be adding the F1 functionality or keywords as that requires programming. I added the description which will help customers locate the topic in organic search. If they do try to use F1 help, they'll be taken to the TOC and they can then search for the topic.

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]
@emyoshi This file needs to be in the API folder with the other Outlook.Folder.* topics.

**Review by emyoshi — COMMENTED** [review]
Thanks for the updates to the metadata. It looks like I can't approve because I originally created the review (which makes sense). I was looking at this page https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request and it's unclear to me whether there is action I am supposed to take. I don't think so (?) but please let me know if that isn't the case.

**Comment by lindalu-MSFT** [comment]
No, you can't approve. That's my job! You needed to move the file to the API folder, right now it is at the root. It needs to be where the other methods are then, then I can add it to the TOC. At this point, let me just take over and get it published correctly. Is that OK with you?

**Comment by emyoshi** [comment]
That would be great - thank you!

I think there will be other changes our team will need to make but I think
it will also be helpful for me to see how/where this is supposed to go
since the others should be relatively similar.

**Comment by emyoshi** [comment]
I *think* I finally figured out how to get the right things to happen *fingers-crossed*

**Pull request closed without merging**